File: Bronze/2562/2562.cpp
// [2562] 최댓값
#include <iostream>
using namespace std;

int main(void)
{
    int max = 0, index = 0, x;
    
    for (int i = 0; i < 9; i++)
    {
        cin >> x;
        if (max < x) {
            max = x;
            index = i+1;
        }
    }
    cout << max << " " << index << "\n";
}

File: Bronze/2562/2562.java
// [2562] 최댓값
import java.util.Scanner;

public class Main {
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        int max = 0, index = 0, x;

        for (int i = 0; i < 9; i++)
        {
            x = sc.nextInt();
            if (max < x) {
                max = x;
                index = i + 1;
            }
        }
        System.out.println(max + " " + index);
    }
}